Apps/Contoso.Android.Puppet/Properties/AndroidManifest.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Not sure what's the _invisible_ change here?

Reply:
Probably https://stackoverflow.com/questions/1772321/what-is-xml-bom-and-how-do-i-detect-it